tutor/src/components/breadcrumb.js

```diff
@@ -25,7 +25,9 @@ class Breadcrumb extends React.Component {
     let crumbClasses, status, title;
     let isCorrect = false;
     let isIncorrect = false;
-    const { step, canReview, isCurrent, className, stepIndex } = this.props;
+    const {
+      step, canReview, isCurrent, className, stepIndex, dataStepIndex,
```

Comment:
I think we can remove `stepIndex` right?

Reply:
no, the stepIndex is still used by the breadcrumb to notify the steps which crumb was clicked.  it's no longer being used for displaying though since using it caused the placeholder steps to be numbered as well

Reply:
ahh, oops 🙈